pkg/resources/management.cattle.io/v3/projectroletemplatebinding/validator_test.go

```diff
@@ -409,78 +409,96 @@ func (p *ProjectRoleTemplateBindingSuite) TestValidationOnUpdate() {
 			allowed: false,
 		},
 		{
-			name: "update previously unset user",
+			name: "update unrelated to previously set service account",
```

Comment:
Nit: This test case effectively unsets the PRTB, which is a bit different than what this name would indicate.